src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -104,9 +125,11 @@ public class FilterEntry
             /// </summary>
             /// <param name="filterName">Filter name, same as host name</param>
             /// <param name="packageManagerSearchViewModel">a reference of the PackageManagerSearchViewModel</param>
-            public FilterEntry(string filterName, PackageManagerSearchViewModel packageManagerSearchViewModel)
+            public FilterEntry(string filterName, string groupName, string tooltip, PackageManagerSearchViewModel packageManagerSearchViewModel)
```

Comment:
@QilongTang This API change is fine for 3.0? Do we need to mark it somewhere when we address others?

Reply:
Ah, would that be API break? I don't think I have a different way of allowing unique `tooltips` for each FilterEntry otherwise ...

Reply:
hi @reddyashish For Dynamo 3.0 we are allowed to break API, that is why I did not mark this one.

Reply:
Actually, I am silly, can we not just introduced another constructor with this signature? And, of course, I could have been setting up the new tooltip property separately after initialization, it's just a bit more code .. .